assets/js/search.js

```diff
@@ -90,6 +90,71 @@ export function search (value) {
   }
 }
 
+export function findIn (elements, matcher) {
```

Comment:
What if we move those functions to `autocomplete/suggestions.js` since they are not related to the search anymore?

Reply:
👍